pkg/sql/explain_plan.go

```diff
@@ -217,7 +217,12 @@ func (e *explainer) leaveNode(name string) {
 }
 
 // subqueryNode implements the planObserver interface.
-func (e *explainer) subqueryNode(_ *subquery) error { return nil }
+func (e *explainer) subqueryNode(sq *subquery) error {
+	if sq.result != nil {
+		e.expr("", "subresult", -1, sq.result)
```

Comment:
Why did this get added? It's not clear to me if this was part of this commit's purpose or if it was just a small fix added in.